admin/class-create-block-theme-admin.php

```diff
@@ -39,7 +39,7 @@ function export_child_theme( $theme ) {
 		$zip = $this->create_zip( $filename );
 
 		$zip = $this->copy_theme_to_zip( $zip );
-		$zip = $this->add_templates_to_zip( $zip, 'current' );
```

Comment:
I was getting errors when I chose the other options, so yeah, maybe pass null

Reply:
It probably would have been wise of me to have actually tested those other options when I changed that instead of assuming that things worked like I thought they would. :P